src/containers/ToDoList/ToDoList.jsx

```diff
@@ -151,6 +161,25 @@ export const ToDoList = () => {
             <ThemeChangingIcon style={{ fontSize: 40, color: "#fff" }} />
           </button>
         </StyledWrapperTitle>
+
+        <StyledWrapperCompletedTasks>
+          <span>Completed tasks</span>
+          <div className="progress-bar">
+            <progress
+              value={numberOfCompletedTasks}
+              max={totalTasksCount}
+              style={{ width: "100%" }}
+            />
```

Comment:
Why not use the `ProgressBar` from the MUI? :-)
https://mui.com/material-ui/react-progress/#linear